docs/semgrep-ci/network-broker.md

```diff
@@ -80,15 +80,15 @@ Ensure that you are logged in to the server where you want to run Semgrep Networ
       ...
   ```
 
-1. Add the generated public key to the Semgrep Cloud Platform:
+1. Add the public key generated in step three to the Semgrep Cloud Platform:
 
    1. Log in to Semgrep Cloud Platform.
    2. Navigate to **Settings** > **Broker**.
-   3. Paste your public key and click **Add Public Key**.
+   3. Paste `YOUR_PUBLIC_KEY` and click **Add Public Key**.
```

Comment:
This seems even more confusing to me. You don't want to paste the string `YOUR_PUBLIC_KEY`, presumably. Do we want to say something like "Paste the generated public key" or "Paste the content of your public key" maybe?

Reply:
"Paste the generated public key" to keep it consistent with the term used on line 83.

Reply:
After going through the doc again, the `YOUR_PUBLIC_KEY` variable is not helpful and is now removed.

I altered all instances of 'the private key' and 'the public key' to be 'your private key' and 'your public key' to keep things consistent and to prevent confusion with Semgrep's private key.